src/server/views/dashboard/partials/dashboard-boxes.njk

```diff
@@ -0,0 +1,18 @@
+{% macro dashboardBox(options) %}
+  <div class="govuk-grid-column-one-third govuk-!-padding-top-3 dashboard-box--{{options.status}}">
+    <strong class="govuk-body govuk-!-font-weight-bold">{{ _.startCase(options.name)}}</strong>
+    {% if options.status === "error" %}
+      <a href="?{{options.queryParam}}=asc" class="govuk-body">{{options.text}}</a>
+    {% else %}
+      <p class="govuk-body">{{options.text}}</p>
+    {% endif %}
+  </div>
+{% endmacro %}
+
+<div class="govuk-grid-row govuk-!-margin-top-2">
+  <div class="govuk-width-container">
+    {% if dashboardBoxes.administrators and req.user.isAdministrator %}
```

Comment:
I tried a few things, but user/admin terminology ended up being too confusing imo. 

I've also changed this so it looks more like bannerToggles (instead of iterating through an array, we just render them based on whether or not the object/property exists).

I'm happy to revert but has the added `and req.user.isAdministrator` benefit. lmk what you think.

Reply:
Nah it looks good. Like the use of marcos as well, wouldn't have thought of doing that 👍